cmd/docker/builder.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strconv"
+
+	pluginmanager "github.com/docker/cli/cli-plugins/manager"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	builderDefaultPlugin     = "buildx"
+	builderDefaultInstallMsg = `To install buildx, see https://docs.docker.com/go/buildx.`
+	builderErrorMsg          = `%s: Required builder component %s is missing or broken.`
```

Comment:
I'd suggest to simplify `buildx` won't change in this context, so no need for the `%s` placeholder (also see other comments)

```suggestion
	builderErrorMsg          = `The buildx component is missing or broken.`
```